dockershrink/package_json.py

```diff
@@ -23,3 +23,23 @@ def get_script(self, name: str) -> Optional[str]:
 
     def raw(self) -> dict:
         return self._raw_data
+
+    def analyze(self) -> dict:
+        """
+        Analyze package.json and return the analysis results.
+        project_info = f
+            # Package name: {package_json_analysis['name']}
```

Comment:
I didn't understand why these lines are inside the docstring?
Is the intention to show what kind of object this method would produce, ie, output format?